source/atlas-cli-changelog.txt

```diff
@@ -12,6 +12,21 @@
    :depth: 1
    :class: singlecol
 
+.. _atlas-cli-1.5.0:
+
+{+atlas-cli+} 1.5.0
+-------------------
+
+Released 9 February 2023
+
+- Updates the :ref:`atlas-config-describe` command to no longer redact 
+  public |api| keys.
+- Updates the default output format for the :ref:`atlas-customDbRoles-list` 
+  command to include inherited roles.
+- Deprecates the :ref:`atlas-integrations-create-NEW_RELIC` command.
```

Comment:
This link won't work when the Atlas CLI release note PR merges because it was removed. We should just refer to it here in monospace text, because the link will break once the page is gone.